src/lint/lint.ts

```diff
@@ -52,18 +51,19 @@ export async function lint(options: Options, config: ProjectConfig) {
     minimumSeverity: Severity.WARNING
```

Comment:
@valdrinkoshi Do you mind linking up the new `filesToIgnore` option for the `WarningFilter`?